ajax/libs/jquery.rondell/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "jquery.rondell",
+  "filename": "jquery.rondell.min.js",
+  "version": "1.1.0",
+  "homepage": "http://projects.sebastianhelzle.net/jquery.rondell/",
+  "keywords": [
+    "carousel",
+    "rondell",
+    "ui",
+    "animate",
+    "slide",
+    "effect"
+  ],
+  "description": "This is a plugin for displaying galeries and other content as carousel, thumbnail gallery, slider and much more!",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Sebobo/jquery.rondell.git"
+  },
+  "author": "Sebastian Helzle",
```

Comment:
Let's add more information about the author according to https://github.com/Sebobo/jquery.rondell/blob/1.1.0/rondell.jquery.json#L5-L8